.github/dependabot.yaml

```diff
@@ -56,6 +56,8 @@ updates:
         update-types: ["version-update:semver-patch"]
       - dependency-name: "github.com/docker/docker"
         versions: ["24.x"]
+      - dependency-name: "github.com/quay/claircore"
+        versions: ["1.5.20"]
```

Comment:
What's the problem with this version? If dependant bumped it, then build and CI have passed. Could you add a comment and explain why it's skipped?

Reply:
added a comment to the file